### PR TITLE
Bug 1989088: Convey proper details for network selection

### PR DESCRIPTION
**Pull request by TimothyAsirJeyasing**
Signed-off-by: Timothy Asir <tjeyasin@redhat.com>

### CODE REVIEW

**Comment by TimothyAsirJeyasing** [comment]
@SanjalKatiyar Please review
BZ:  https://bugzilla.redhat.com/show_bug.cgi?id=1989088

**Comment by TimothyAsirJeyasing** [comment]
Screenshot from 2023-01-03 15-11-48
Screenshot from 2023-01-03 15-11-41

**Comment by SanjalKatiyar** [comment]
@TimothyAsirJeyasing add BZ to the PR...

**Comment by SanjalKatiyar** [comment]
run i18n command...

**Comment by SanjalKatiyar** [comment]
> run i18n command...

also add BZ to the title...

File: packages/odf/components/create-storage-system/create-storage-system-steps/security-and-network-step/configure.tsx [hunk 1]
```diff
@@ -107,30 +107,39 @@ export const NetworkFormGroup: React.FC<NetworkFormGroupProps> = ({
...
           value={NetworkType.DEFAULT}
           id={NetworkType.DEFAULT}
         />
         <Radio
           isChecked={networkType === NetworkType.MULTUS}
           name="custom-network"
-          label={t('Custom (Multus)')}
+          label={
+            <>
+              {t('Custom (Multus')}
```
Comment (SanjalKatiyar) [inline packages/odf/components/create-storage-system/create-storage-system-steps/security-and-network-step/configure.tsx:135]:
```suggestion
              {t('Custom (Multus)')}
```

Comment (TimothyAsirJeyasing) [reply packages/odf/components/create-storage-system/create-storage-system-steps/security-and-network-step/configure.tsx:135]: Done

**Pull request merged**